apps/office-addin/src/content/components/App.tsx

```diff
@@ -36,10 +36,27 @@ export default function App({ isOfficeInitialized, newlyInserted }: AppProps) {
     }
   }
   return (
-    <div className="m-10">
-      <H1 className={{ root: "text-xl text-red-900" }}>KlickerUZH: PowerPoint Add-in</H1>
-      <img src="assets/logo-filled.png" alt="KlickerUZH Logo" className="my-12 ml-auto mr-auto h-72 w-72"></img>
-      <URLForm slideID={slideID} />
+    <div className="font-sans">
+      <div className="flex flex-row bg-slate-100 p-2 gap-4 items-center">
+        <img src="assets/logo-filled.png" alt="KlickerUZH Logo" className="h-16 w-16"></img>
+        <H1 className={{ root: "" }}>KlickerUZH: Embed Evaluation</H1>
+      </div>
+      <div className="p-4">
+        <div className="flex flex-row gap-4 mb-4">
+          <div className="flex-1">
+            <ol className="list-decimal list-inside">
+              <li>Go to https://manage.klicker.uzh.ch/sessions</li>
```

Comment:
deliberately not a "link" so that people do not get redirected to the klicker page within PP